tests/renderer/remote-loader-spec.ts

```diff
@@ -238,24 +239,43 @@ describe('RemoteLoader', () => {
       expect(store.channelsToShow).toContain(ElectronReleaseChannel.beta);
     });
 
-    it('does not load unsupported versions of Fiddle', async () => {
+    it('tries to download missing versions of Electron', async () => {
+      const version = '5.0.0';
       instance.getPackageVersionFromRef = jest
         .fn()
-        .mockReturnValueOnce('5.0.0');
+        .mockReturnValueOnce(version);
+      instance.askToDownloadMissingVersion = jest
+        .fn()
+        .mockReturnValueOnce(true);
+
+      const result = await instance.setElectronVersionWithRef(version);
+      expect(result).toBe(true);
+      expect(store.addNewVersions).toBeCalledWith([
+        {
+          source: VersionSource.remote,
+          state: VersionState.unknown,
+          version,
+        },
+      ]);
+      expect(store.setVersion).toBeCalledWith(version);
+    });
+
+    it('does not load unsupported versions of Electron', async () => {
```

Comment:
Could these two tests be renamed to better reflect what they're testing?

I think I'm reading this right that it's not checking to see if $version is supported, but whether or not the result of `askToDownload` is honored